### PR TITLE
[Dot-3170] Handled non image files 

### PR DESCRIPTION
**Pull request by parthlambdatest**

### CODE REVIEW

File: src/commander/upload.ts [hunk 1]
```diff
@@ -31,6 +32,11 @@ command
             return;
         }
 
+        if (path.extname(directory).toLowerCase() === '.zip') {
```
Comment (pinanks) [inline src/commander/upload.ts:35]: Use constants for different extensions

File: src/lib/screenshot.ts [hunk 2]
```diff
@@ -165,8 +165,19 @@ function getImageDimensions(filePath: string): { width: number, height: number }
     return null;
 }
 
+function isImage(buffer: Buffer): boolean {
+    const magicNumbers = [
+        { ext: 'jpg', magic: Buffer.from([0xFF, 0xD8, 0xFF]) },
```
Comment (pinanks) [inline src/lib/screenshot.ts:170]: Use constants

File: src/lib/screenshot.ts [hunk 3]
```diff
@@ -6,6 +6,12 @@ import * as utils from "./utils.js"
 import constants from './constants.js'
 import chalk from 'chalk';
 
+const magicNumbers = [
+    { ext: 'jpg', magic: Buffer.from([0xFF, 0xD8, 0xFF]) },
```
Comment (pinanks) [inline src/lib/screenshot.ts:10]: These as well

File: src/lib/screenshot.ts [hunk 4]
```diff
@@ -195,18 +209,26 @@ export async function uploadScreenshots(ctx: Context): Promise<void> {
...
                 }
             }
         }
     }
 
     await processDirectory(ctx.uploadFilePath);
+    if(noOfScreenshots == 0){
+        console.log(`No screenshots uploaded.`);
+    } else {
+    console.log(`${noOfScreenshots} screenshots uploaded successfully.`);
```
Comment (sushobhit-lt) [inline src/lib/screenshot.ts:232]: use a formatter here

**Review by sushobhit-lt — APPROVED** [review]

File: src/lib/screenshot.ts (hunk 4 above)
Comment (pinanks) [reply src/lib/screenshot.ts]: use ctx.log

**Pull request merged**